SSA/Projects/InstCombine/LLVM/CLITests.lean

```diff
@@ -187,34 +187,45 @@ def CliTest.eval (test : CliTest) (values : Vector ℤ test.context.length) (hMv
    concrete_test.eval values'
 -/
 
-def InstCombine.mkValuation (ctxt : MContext 0) (values : Vector Int ctxt.length): Ctxt.Valuation ctxt :=
+def InstCombine.mkValuation (ctxt : MContext 0) (values : Vector (Option Int) ctxt.length): Ctxt.Valuation ctxt :=
 match ctxt, values with
   | [], ⟨[],_⟩ => Ctxt.Valuation.nil
   | ty::tys, ⟨val::vals,hlen⟩ =>
-    let valsVec : Vector ℤ tys.length := ⟨vals,by aesop⟩
+    let valsVec : Vector (Option ℤ) tys.length := ⟨vals,by aesop⟩
     let valuation' := mkValuation tys valsVec
-    let newTy : toType ty := ↑(val)
-    Ctxt.Valuation.snoc valuation' newTy
+    match ty with
+      | .bitvec (.concrete w) =>
+         let newTy : toType (.bitvec (.concrete w)) := Option.map (Std.BitVec.ofInt w) val
+         Ctxt.Valuation.snoc valuation' newTy
```

Comment:
We should be consistent about using `Int` or `ℤ`. At least within one function, but it might make sense to agree on a preference for the whole project. I don't have strong opinions about one or the other.